src/pages/plots.jsx

```diff
@@ -65,6 +77,13 @@ const Plots = (props) => {
                 availablePlots={props.environmentPlots}
               />
             ) : null}
+            {tab === AEROSOLS ? (
```

Comment:
```suggestion
            {tab === AEROSOLS && PART_MC ? (
```

Need to wrap in feature flag